examples/integration/packages/my-integration/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "@example/my-integration",
+    "private": true,
+    "version": "0.0.1",
+    "type": "module",
+    "exports": {
+      ".": "./index.ts",
+      "./components": "./components.ts"
```

Comment:
I couldn't find a way to export the components from root as well, it blows up when the integration is imported in `astro.config.mjs`

Any magic to make this work easily so that we don't need to use the `@example/my-integration/components` import contvention?

Reply:
I think it's worth removing components from the example - when looking through it, I was trying to find where the integration was linked to the component. We already have the updated `component` example to show how to export components.

Reply:
I think I agree — I wouldn’t include components in this starter necessarily as integrations that add components don’t seem to be the norm? Gives us some freedom to show of the integration API more specifically.

Reply:
I was leaning this way as well, glad to hear you both agree!  I'll pull out the components and keep this to just an example of the integration API itself :+1: